ppr-ui/src/composables/mhrInformation/useMhrInformation.ts

```diff
@@ -353,7 +353,7 @@ export const useMhrInformation = () => {
       declaredValue: getMhrTransferDeclaredValue.value,
       consideration: getMhrTransferConsideration.value,
       transferDate: getMhrTransferDate.value,
-      ownLand: getMhrTransferOwnLand.value || null,
+      ownLand: getMhrTransferOwnLand.value,
```

Comment:
Totally!
I think in the Registration flow, null was set to determine no-entry-made when the user navigated to the Review Confirm step. 

In the Transfer flow, it's required to navigate to review confirm, with no indeterminate state ie non selection state etc, and the schema doesn't like those `null` properties when types are enforced.

If it was an optional property (i didn't check), it would have to be omitted from the payload vs a null value.

Reply:
Thanks for clarification. I recall this was a mandatory in at least one of the flows... maybe Roy would have some input.